src/controllers/article.js

```diff
@@ -127,14 +127,13 @@ class ArticleController {
         .hasBeenFavorited(getOneArticle.id, request.userData.payload.id)
         : false;
 
-      const articleRatingStar = await articleAverageRating(request);
+      getOneArticle.articleRatingStar = await articleAverageRating(request);
```

Comment:
Nice one here. @marcdomain love the way you made the article average rating a property articleRatingStar.